Examples/java/src/main/java/software/amazon/cryptography/example/hierarchy/CreateKeyExample.java

```diff
@@ -92,8 +94,45 @@ public static String CreateKey(
           .build()
       )
       .Identifier();
-
     assert actualBranchKeyId.equals(branchKeyId);
+    if (
+      hierarchyVersion == HierarchyVersion.v1 ||
+      kmsKeyArn != Fixtures.KMS_KEY_FOR_HV2_ONLY
+    ) {
+      return branchKeyId;
+    }
+    // HV2 sends the encryption context without any transformation.
+    // We have a kms key `Fixtures.KMS_KEY_FOR_HV2_ONLY`, that requires EC to be exactly {"Robbie": "Is a Dog."} in its key policy.
+    // For demostration, we will create a key with a different EC then the one that is expected and see it fail.
+    final Map<String, String> encryptionContextFailingCase =
+      Collections.singletonMap("I", "am not a Dog.");
+    boolean exceptionThrown = false;
+    try {
+      _admin
+        .CreateKey(
+          CreateKeyInput
+            .builder()
+            // This is the KMS ARN that will be used to protect the Branch Key.
+            // It is a required argument.
+            .KmsArn(KmsSymmetricKeyArn.builder().KmsKeyArn(kmsKeyArn).build())
+            // If you need to specify the Identifier for a Branch Key, you may.
+            // This is an optional argument.
+            .Identifier(branchKeyId)
+            // If a branch key Identifier is provided,
+            // custom encryption context MUST be provided as well.
+            .EncryptionContext(encryptionContextFailingCase)
+            // The Branch Key Store Admin can create HV-1 or HV-2 Branch Keys
+            .HierarchyVersion(_hierarchyVersion)
+            // But the Strategy MUST support the Hierarchy Version
+            .Strategy(strategy)
+            .build()
+        )
+        .Identifier();
+    } catch (KmsException e) {
+      exceptionThrown = true;
+    }
+    assert exceptionThrown;
```

Comment:
Blocking: Just move all of this to a Test.
Do not keep it in line with the examples, that confuses customers, and complicates this `CreateKey` method that we use through out the Example Testing.